README.md

```diff
@@ -94,6 +97,183 @@ see [Developer’s guide to auth with Azure Resource Manager API](http://aka.ms/
 After creating the service principal, you should have three pieces of information, a client id (GUID), client secret
 (string) and tenant id (GUID) or domain name (string).
 
+# Azure Multiple API versions & Profiles
+
+With 0.15.0 of Azure SDK, multiple API versions and profiles are introduced. With these changes, each individual gem
+has multiple versions of the services and several profiles. The azure sdk rollup gem also consists of several profiles. The following section provides details on the usage of multiple API versions and profiles.
+
+## Why Multiple API versions?
+
+With the previous Azure SDK, you will have access to the latest (at the time of release) versions of our services. With 
```

Comment:
Suggestions on rewording: "Versions 0.14.0 and older, would have access to the latest versions of Azure services at the time of release. Each release up to 0.14.0, would include the latest available api-version of the services. There wasn't an option to use an older api-version of the service (except for using an older version of the sdk gem). With the introduction of multiple API versions per gem, any api-version available for the service can be explicitly targeted."
@salameer @sarangan12

Reply:
Done